qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -106,3 +106,16 @@
             # reset media player play mode
             post_cmd += "reg add HKCU\Software\Microsoft\MediaPlayer\Preferences /v ModeLoop /t REG_DWORD /d 0 /f"
             target_process = wmplayer.exe
+        - with_rng:
+            driver_name = viorng
+            run_bgstress = rng_bat
+            target_process = D:\random_%PROCESSOR_ARCHITECTURE%.exe
+            session_cmd_timeout = 240
+            read_rng_cmd  = "X:\random_%PROCESSOR_ARCHITECTURE%.exe"
+            rng_data_rex = "0x\w"
+            driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
```

Comment:
I think also need update driver_id_cmd.
driver_id_cmd = C:\devcon.exe find * | find "VirtIO"

If some drive if not installed, test will be failed.

17:40:03 ERROR| TestFail: Didn't find driver info from guest PCI\VEN_1AF4&DEV_1000&SUBSYS_00011AF4&REV_00\3&13C0B0C5&0&28: Red Hat VirtIO Ethernet Adapter
17:40:03 ERROR| PCI\VEN_1AF4&DEV_1001&SUBSYS_00021AF4&REV_00\3&13C0B0C5&0&20: Red Hat VirtIO SCSI controller
17:40:03 ERROR| SCSI\DISK&VEN_RED_HAT&PROD_VIRTIO\4&35110308&0&000000       : Red Hat VirtIO SCSI Disk Device